apps/analog-app/src/app/pages/(home).page.ts

```diff
@@ -51,17 +51,7 @@ export const routeMeta: RouteMeta = {
   ],
 })
 export default class ProductListComponent {
-  products!: Product[];
-  http = inject(HttpClient);
-
-  ngOnInit() {
-    this.http
-      .get<Product[]>('/api/v1/products')
-      .pipe(catchError(() => of([])))
-      .subscribe((products) => {
-        this.products = products;
-      });
-  }
+  data = toSignal(useLoad<Load>());
```

Comment:
```suggestion
  data = toSignal(useLoad<Load>(), { requireSync: true });
```
We can avoid the force unwrap if we tell Angular to immediately sync the Observable with the signal.

Reply:
Ahh, thanks